DependencyInjection/DoctrineExtension.php

```diff
@@ -902,17 +942,17 @@ protected function loadCacheDriver($cacheName, $objectManagerName, array $cacheD
      *
      * @param array<string, mixed> $entityManager A configured ORM entity manager.
      */
-    protected function loadOrmCacheDrivers(array $entityManager, ContainerBuilder $container)
+    protected function loadOrmCacheDrivers(array $entityManager, ContainerBuilder $container, bool $usePsr6)
```

Comment:
How come $usePsr6 was made optional in one protected function and non-optional here?